src/test/java/de/tum/in/www1/artemis/service/exam/StudentExamAccessServiceTest.java

```diff
@@ -64,37 +64,53 @@ void init() {
     @Test
     @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
     void testIsAtLeastStudentInCourse() {
-        assertThrows(AccessForbiddenException.class, () -> studentExamAccessService.checkCourseAndExamAccessElseThrow(course2.getId(), exam2.getId(), student1, false, true));
-        assertThrows(AccessForbiddenException.class, () -> studentExamAccessService.checkStudentExamAccessElseThrow(course2.getId(), exam2.getId(), studentExam1.getId()));
-        assertThrows(AccessForbiddenException.class, () -> studentExamAccessService.checkStudentExamAccessElseThrow(course2.getId(), exam2.getId(), studentExam1, student1));
+        assertThatExceptionOfType(AccessForbiddenException.class)
+                .isThrownBy(() -> studentExamAccessService.checkCourseAndExamAccessElseThrow(course2.getId(), exam2.getId(), student1, false, true));
+
+        assertThatExceptionOfType(AccessForbiddenException.class)
+                .isThrownBy(() -> studentExamAccessService.checkStudentExamAccessElseThrow(course2.getId(), exam2.getId(), studentExam1.getId()));
+
+        assertThatExceptionOfType(AccessForbiddenException.class)
+                .isThrownBy(() -> studentExamAccessService.checkStudentExamAccessElseThrow(course2.getId(), exam2.getId(), studentExam1, student1));
     }
 
     @Test
     @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
     void testExamExists() {
-        assertThrows(EntityNotFoundException.class, () -> studentExamAccessService.checkCourseAndExamAccessElseThrow(course1.getId(), -1L, student1, false, true));
-        assertThrows(EntityNotFoundException.class, () -> studentExamAccessService.checkStudentExamAccessElseThrow(course1.getId(), -1L, studentExam1.getId()));
-        assertThrows(EntityNotFoundException.class, () -> studentExamAccessService.checkStudentExamAccessElseThrow(course1.getId(), -1L, studentExam1, student1));
+        assertThatExceptionOfType(EntityNotFoundException.class)
```

Comment:
Probably add new lines for consistency. Or remove them in other places.

Reply:
fixed